extensions/notebook/package.json

```diff
@@ -282,6 +299,10 @@
         {
           "command": "notebook.command.searchUntitledBook",
           "when": "false"
+        },
+        {
+          "command": "notebook.command.trustBook",
+          "when": "false"
```

Comment:
You don't want to have this be available in the command palette?

Reply:
Why not? That seems like a reasonable thing to have

Reply:
No reason in particular. Have made the change.